index.html

```diff
@@ -3,9 +3,34 @@
   <head>
     <meta charset="utf-8">
     <title>super-fantastic-dashboard</title>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <style>
+      body {
+        background-color: lightblue;
+        background-repeat: no-repeat;
+        background-position: center center;
+        background-attachment: fixed;
+        background-size: cover;
+      }
+    </style>
   </head>
   <body>
     <div id="app"></div>
     <script src="/dist/build.js"></script>
+    <script>
```

Comment:
Curious what Axios is doing here for us - what is the GET url that we're constructing? 

Can this be done using a simple AJAX request and assign the response to a variable and then passing that variable to a function? While it makes it easier - I just want us to think about tradeoffs - and having unnecessary dependencies. Considering this is the one and only place we're making a 3rd party request, it may be more worth it for now to build out our own _if_ it's simple enough. Happy to be wrong also.